pbrd/pbr_vty.c

```diff
@@ -199,10 +199,9 @@ DEFPY(pbr_map_match_ip_proto, pbr_map_match_ip_proto_cmd,
 		return CMD_WARNING_CONFIG_FAILED;
 
 	if (!no) {
-		p = getprotobyname(ip_proto);
-		if (!p) {
+		if (!ip_proto || !(p = getprotobyname(ip_proto))) {
```

Comment:
Why should we report a warning? We should just unset any `ip-protocol` command and that's it.

Reply:
I only modified the "set" case:
```
(config-pbr-map)# match ip-protocol 
Unable to convert (null) to proto id
(config-pbr-map)# match ip-protocol udp 
(config-pbr-map)# no match ip-protocol
(config-pbr-map)# 
```

Reply:
Ah, sorry, didn't check the code (expanded) :1st_place_medal: